tests/contrib/trace/test_plugin.py

```diff
@@ -49,3 +49,87 @@ def test__trace_plugin__valid_schema(mock_send_report, handler_with_trace, mock_
             'projectId',
             'plugins.uploads',
         ])
+
+
+@mock.patch('iopipe.report.send_report', autospec=True)
+def test__trace_plugin__auto_http__http(mock_send_report, handler_with_trace_auto_http, mock_context):
+    iopipe, handler = handler_with_trace_auto_http
+
+    assert len(iopipe.config['plugins']) == 1
+
+    handler({}, mock_context)
+
+    metrics = iopipe.report.report['custom_metrics']
+    traces = iopipe.report.report['performanceEntries']
+
+    assert len(traces) == 6
+    assert len(metrics) == 50
+    assert all([m['name'].startswith('@iopipe/trace.') for m in metrics])
+    assert any([m['s'] == 'autoHttp'
+                for m in metrics
+                if m['name'].startswith('@iopipe/trace.') and m['name'].endswith('.type')])
+
+
+@mock.patch('iopipe.report.send_report', autospec=True)
+def test_trace_plugin__auto_http__https(mock_send_report, handler_with_trace_auto_https, mock_context):
+    iopipe, handler = handler_with_trace_auto_https
+
+    assert len(iopipe.config['plugins']) == 1
+
+    handler({}, mock_context)
+
+    metrics = iopipe.report.report['custom_metrics']
+    traces = iopipe.report.report['performanceEntries']
+
+    assert len(traces) == 3
+    assert len(metrics) == 25
+    assert all([m['name'].startswith('@iopipe/trace.') for m in metrics])
+    assert any([m['s'] == 'autoHttp'
+                for m in metrics
+                if m['name'].startswith('@iopipe/trace.') and m['name'].endswith('.type')])
+
+    id = next((m['name'].split('.', 2)[1] for m in metrics
+               if m['name'].endswith('.type')))
+    metric_map = {m['name']: m.get('s', m.get('n')) for m in metrics}
+    expected_key_value = {
+        '@iopipe/trace.%s.request.hash' % id: '',
+        '@iopipe/trace.%s.request.headers.Accept' % id: '*/*',
```

Comment:
I think Node is returning headers in lower case when working with the `http` library. What do you think about coercing to lower case? I'd like to be consistent across libs for easy DX in consumers.